test/lib/localStorage.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * A simple localStorage implementation using in-memory storage.
+ */
+export default class LocalStorage {
+  constructor(private storageObject) { }
+
+  getItem (key: string) {
+    return this.storageObject[key];
+  }
+
+  setItem (key: string, value: any) {
+    this.storageObject[key] = value;
+  }
+
+  removeItem (key: string) {
```

Comment:
hmm not sure of the purpose of this class is - you could easily use a `Map` instead of this

Reply:
This is copied from SNJS. I believe the goal is so that the calling class can use the localStorage API-syntax without actually using localStorage, but using a memory store.